backends/ebpf/CMakeLists.txt

```diff
@@ -221,6 +221,36 @@ else()
   set (SUPPORTS_KERNEL False)
 endif()
 
+# List of executable dependencies
+set(PROGRAMS scapy tcpdump pcap)
+
+# List of library dependencies
+set(LIBRARIES libpcap-dev gcc-multilib)
```

Comment:
If this is going to be used in multiple backends, will these names `PROGRAMS` and `LIBRARIES` conflict between different backends that want to set them to different things?

Reply:
I have updated these names with TEST_DEPENDENCY_PROGRAMS and  TEST_DEPENDENCY_LIBRARIES to avoid any conflict